Makefile

```diff
@@ -56,6 +63,9 @@ integration-test: depend  ## runs integration tests, defined as tests which requ
 .PHONY: lint
 lint: vet verify-boilerplate verify-helm-docs
 
+.PHONY: verify
+verify: depend build test ## tests and builds trust-manager
```

Comment:
should CI not run `verify-boilerplate` and `verify-helm-docs` too?

Reply:
It does, I think! `verify` depends on `test` which depends on `lint`, which runs both `verify-boilerplate` and `verify-helm-docs`

Reply:
Ah! yes thank you